Civi/Api4/Generic/AbstractAction.php

```diff
@@ -35,6 +36,8 @@
  */
 abstract class AbstractAction implements \ArrayAccess {
 
+  use MagicGetterSetterTrait;
```

Comment:
Nitpick: I dislike the way PHPStorm automatically generates duplicate `use` statements when you use a trait. The one on line 17 could be moved here, and the same outcome would be achieved with one line instead of 2.

Reply:
Updated